fe-games/src/index.css

```diff
@@ -91,19 +91,6 @@ justify-content: left;
 .nav_link{
 justify-content: right;
 }
-.loader {
-  border: 16px solid #f3f3f3;
-  border-top: 16px solid #3498db;
-  border-radius: 50%;
-  width: 130px;
-  height: 130px;
-  animation: spin 2s linear infinite;
-}
-
-@keyframes spin {
-  0%  { transform: rotate(0deg); }
-  100% { transform: rotate(360deg); }
-}
```

Comment:
This app should be designed for mobile first. When you view this in the browser, it isn't responsive and the text boxes are out of line. The text colour is difficult to read, please change this so it provides a better experience for the user 👍